data/json/items/tool_armor.json

```diff
@@ -1861,6 +1910,102 @@
     "ammo": "gasfilter_m",
     "use_action": [ "GASMASK" ]
   },
+  {
+    "id": "mask_gas_m50",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "M50 gas mask" },
+    "category": "clothing",
+    "description": "A full-face gas mask rated for use in chemical, biological, radiological, and nuclear-contaminated areas, built from a comfortable rubber-silicone blend and featuring a panoramic eyepiece, drinking tube, and a communications interface.  The standard gasmask of the United States military, this respirator possesses a twin-filter system incompatible with standard 40-mm NATO filters.  It must be prepared before use.",
+    "flags": [ "SLEEP_IGNORE" ],
+    "weight": "860 g",
+    "volume": "1 L",
+    "longest_side": "305 mm",
+    "price": 81400,
+    "//": "Retail price, believe it or not. Sourced from https://www.galls.com/avon-fm50-full-face-mask-system",
```

Comment:
~~Is this an attempt to get me to buy one~~

Reply:
Little known fact, I am indeed paid a commission by Avon for every gas mask they sell cause of these changes. Get to buying, people. Over 800$ for a gas mask is good money.